src/librustc_session/lint.rs

```diff
@@ -195,7 +195,7 @@ pub enum BuiltinLintDiagnostics {
 }
 
 /// Lints that are buffered up early on in the `Session` before the
-/// `LintLevels` is calculated. These are later passed to `librustc_middle`.
```

Comment:
The struct was moved from libsyntax to librustc_session and lint mechanism has been changed so I think we can remove the sentence.